src/backend/XmlHttpRequest.ts

```diff
@@ -97,6 +104,7 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
 
   public readonly prefixUrl: string;
   private _index: FileIndex<{}>;
+  private _useFetch: boolean;
```

Comment:
As in previous comments, it would be cleaner to replace this with function properties for downloading files and getting their size.